packages/itwin/breakdown-trees/package.json

```diff
@@ -41,81 +41,86 @@
     "i18next": "^10.2.2"
   },
   "devDependencies": {
-    "@itwin/core-bentley": "^3.0.0",
+    "@itwin/appui-abstract": "^3.0.0",
+    "@itwin/appui-layout-react": "^3.0.0",
+    "@itwin/appui-react": "^3.0.0",
     "@itwin/build-tools": "^3.0.0",
-    "@itwin/core-geometry": "^3.0.0",
+    "@itwin/components-react": "^3.0.0",
+    "@itwin/core-bentley": "^3.0.0",
     "@itwin/core-common": "^3.0.0",
     "@itwin/core-frontend": "^3.0.0",
+    "@itwin/core-geometry": "^3.0.0",
     "@itwin/core-i18n": "^3.0.0",
+    "@itwin/core-markup": "^3.0.0",
+    "@itwin/core-orbitgt": "^3.0.0",
+    "@itwin/core-quantity": "^3.0.0",
+    "@itwin/core-react": "^3.0.0",
+    "@itwin/eslint-plugin": "^3.0.0",
+    "@itwin/imodel-components-react": "^3.0.0",
+    "@itwin/measure-tools-react": "workspace:*",
     "@itwin/presentation-common": "^3.0.0",
     "@itwin/presentation-components": "^3.0.0",
     "@itwin/presentation-frontend": "^3.0.0",
-    "@itwin/eslint-plugin": "^3.0.0",
-    "@itwin/appui-abstract": "^3.0.0",
-    "@itwin/components-react": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
-    "@itwin/appui-react": "^3.0.0",
-    "@itwin/appui-layout-react": "^3.0.0",
-    "@itwin/measure-tools-react": "workspace:*",
+    "@itwin/webgl-compatibility": "^3.0.0",
     "@testing-library/react": "^11.2.6",
-    "@types/i18next": "^8.4.2",
-    "@types/react": "^17.0.19",
-    "@types/react-dom": "^17.0.9",
-    "@types/mock-require": "^2.0.0",
-    "mock-local-storage": "^1.1.8",
-    "mock-require": "^3.0.3",
-    "@types/faker": "^4.1.0",
-    "@types/sinon": "^7.5.1",
-    "@types/mocha": "^8.2.2",
     "@types/chai": "4.2.11",
     "@types/chai-as-promised": "^7.1.2",
     "@types/chai-jest-snapshot": "^1.3.4",
     "@types/chai-spies": "^1.0.0",
     "@types/enzyme": "3.10.5",
+    "@types/faker": "^4.1.0",
+    "@types/i18next": "^8.4.2",
+    "@types/mocha": "^8.2.2",
+    "@types/mock-require": "^2.0.0",
+    "@types/react": "^17.0.19",
+    "@types/react-dom": "^17.0.9",
     "@types/react-redux": "^7.1.9",
+    "@types/sinon": "^7.5.1",
     "@typescript-eslint/eslint-plugin": "^5.3.1",
-    "eslint": "^7.11.0",
-    "react-redux": "^7.2.0",
-    "redux": "^4.0.4",
-    "mocha": "^8.3.2",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.3",
     "chai": "^4.2.0",
-    "nyc": "^15.1.0",
     "chai-as-promised": "^7.1.1",
     "chai-jest-snapshot": "^2.0.0",
     "chai-spies": "1.0.0",
-    "sinon": "^7.5.0",
-    "sinon-chai": "^3.2.0",
-    "faker": "^4.1.0",
+    "cpx": "^1.5.0",
```

Comment:
Should not use cpx.
Replace by "cpx2": "^3.0.0",

Reply:
snuck that into https://github.com/iTwin/viewer-components-react/pull/180